python/cudf/cudf/core/dataframe.py

```diff
@@ -5562,6 +5563,12 @@ def from_pandas(cls, dataframe, nan_as_null=no_default):
         elif hasattr(dataframe, "__dataframe__"):
             # TODO: Probably should be handled in the constructor as
             # this isn't pandas specific
+            assert version.parse(cudf.__version__) < version.parse("25.04.00")
+            warnings.warn(
```

Comment:
Do we need to filter this warning too?

Reply:
We don't because that branch wasn't tested before :upside_down_face: I thought about it but then skipped it since it's already untested and we're removing this code next release anyway.

Reply:
Ok cool thanks, LGTM